indexer/altitude_loader.cpp

```diff
@@ -54,7 +54,7 @@ AltitudeLoader::AltitudeLoader(MwmValue const & mwmValue)
   }
 }
 
-bool AltitudeLoader::HasAltitudes() const { return m_header.m_minAltitude != kInvalidAltitude; }
+bool AltitudeLoader::HasAltitudes() const { return m_reader.get() != nullptr; }
```

Comment:
In case of an empty mwm (an mwm without features) the default constructor of AltitudeHeader will be called. It means m_header.m_minAltitude will be initialized with AltitudeHeader::Reset() method:
```c++
  void Reset()
  {
    m_version = 0;
    m_minAltitude = kDefaultAltitudeMeters;
    m_featureTableOffset = 0;
    m_altitudesOffset = 0;
    m_endOffset = 0;
  }
```
So in case of mwm without features m_minAltitude == kDefaultAltitudeMeters == 0. As we agreed previously kInvalidAltitude is mostly used on generation time.